### PR TITLE
BACKLOG-10211 style the image cover in MediaPicker field

### PR DESCRIPTION
**Pull request by Taybou**

### CODE REVIEW

File: src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker_filled.jsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ const styles = theme => ({
         width: '100%',
         height: '100%',
         padding: '2px',
+        objectFit: 'contain',
```
Comment (FBerthelot) [inline src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker_filled.jsx]: not working on IE

Comment (Taybou) [reply src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker_filled.jsx]: OOOOH MINCE, nice catch

Thank you

File: src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker_filled.jsx [hunk 2]
```diff
@@ -71,10 +74,9 @@ const MediaPickerFilledCmp = ({t, field, selectedImgId, classes}) => {
     return (
         <article className={classes.imageSelectedContainer} data-sel-media-picker="filled">
             <div className={classes.imageSelectedImgContainer}>
-                <img
+                <figure
```
Comment (FBerthelot) [inline src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker_filled.jsx]: 👍

File: src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker_filled.jsx [hunk 3]
```diff
@@ -24,15 +24,18 @@ const styles = theme => ({
         height: theme.spacing.unit * 9,
         width: theme.spacing.unit * 9,
         overflow: 'hidden',
-        display: 'flex',
+        display: 'inline-block',
+        padding: '2px',
         alignItems: 'center',
```
Comment (FBerthelot) [inline src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker_filled.jsx]: a supprimer si tu n'es plus en flex.

Comment (Taybou) [reply src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker_filled.jsx]: Yes, of course ^^

**Pull request merged**